SmartEVSE-3/include/evse.h

```diff
@@ -30,6 +30,10 @@
 #define DBG 0  //comment or set to 0 for production release, 0 = no debug 1 = debug over telnet, 2 = debug over usb AND telnet
 #endif
 
+#ifndef MQTT_DISABLED
+// #define MQTT_DISABLED 1                                                      // Uncomment to disable MQTT support in code
```

Comment:
Please: 1,$s/ifndef MQTT_DISABLED/ifdef MQTT/g

and change //#define MQTT_DISABLED to:
#define MQTT 1 //comment or set to 0 to disable MQTT

...so it is more in line with the other compile options.